### PR TITLE
Generated v0.1.0 release

### PR DESCRIPTION
**Pull request by ellis-bigelow**
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/kfserving/210)

### CODE REVIEW

**Comment by johnugeorge** [comment]
Isn't it v0.1.0 release?

**Comment by ellis-bigelow** [comment]
> Isn't it v0.1.0 release?

Yes! I just misnamed the issue :(

File: hack/generate-install.sh [hunk 1]
```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
```
Comment (yuzisun) [inline hack/generate-install.sh]: should we follow the knative release way to upload this yaml to the release artifact?

File: install/v0.1.0/kfserving.yaml [hunk 2]
```diff
@@ -0,0 +1,633 @@
+apiVersion: v1
```
Comment (yuzisun) [inline install/v0.1.0/kfserving.yaml]: should we follow the knative release way to upload this yaml to the GitHub release artifact?

Comment (ellis-bigelow) [reply install/v0.1.0/kfserving.yaml]: Yeah -- once these are checked in, I'll absolutely add them as release artifacts.

File: install/v0.1.0/kfserving.yaml [hunk 3]
```diff
@@ -0,0 +1,633 @@
...
+        command:
+        - /manager
+        env:
+        - name: POD_NAMESPACE
+          valueFrom:
+            fieldRef:
+              fieldPath: metadata.namespace
+        - name: SECRET_NAME
+          value: kfserving-webhook-server-secret
+        image: gcr.io/kfserving/kfserving-controller:latest
```
Comment (hougangliu) [inline install/v0.1.0/kfserving.yaml]: I think we'd better rename `gcr.io/kfserving/kfserving-controller:latest` to `gcr.io/kfserving/kfserving-controller:v0.1.0`

Comment (animeshsingh) [reply install/v0.1.0/kfserving.yaml]: Infact we should, given this a release, we should not be pulling from master

Comment (ellis-bigelow) [reply install/v0.1.0/kfserving.yaml]: Wow this completely skipped my mind. Thank you!

**Pull request merged**